data/json/mapgen/shelter.json

```diff
@@ -388,6 +503,219 @@
       "palettes": [ { "distribution": [ [ "shelter_used", 10 ], [ "shelter_vandal", 10 ] ] } ]
     }
   },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "weight": 70,
+    "//": "Copy of shelter_2_vandal, unfinished before the cataclysm. No roof.",
+    "om_terrain": [ "shelter_2b_vandal" ],
+    "object": {
+      "fill_ter": "t_floor",
+      "rows": [
+        "                        ",
+        "                        ",
+        "                        ",
+        "                 ~~~~   ",
+        "                 ~~~~   ",
+        "                        ",
+        "                        ",
+        "   ||--              |  ",
+        "   |*|..                ",
+        "   |....                ",
+        "  |--|..                ",
+        "  |......               ",
+        "  :.......              ",
+        "  |.............        ",
+        "  |.......|--|.......|  ",
+        "  |........>>........|  ",
+        "  |........>>........|  ",
+        "  :.......|--|.......:  ",
+        "  |..................|  ",
+        "  |..................|  ",
+        "  |--..............--|  ",
+        "   |...............c|4  ",
+        "   |--------|......c|   ",
+        "            |-:-+-:-|   "
+      ],
+      "palettes": [ "shelter_vandal" ],
+      "terrain": { "~": "t_dirt" },
+      "items": { "~": { "item": "lumberyard", "chance": 90, "repeat": [ 3, 5 ] } }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "weight": 50,
+    "//": "Infested spawn for Compromized Shelter scenario - copy of shelter_1. Window on right wall near basement door removed for scenario balance, sorry. Spawns are a bit sporadic, but usually give at least 1 running enemy and ~4-7 others at 0.6 density.",
```

Comment:
In this case, I removed the window because it was two tiles away from the basement door and it became very easy to completely avoid confrontation and just run right outside (making the scenario basically just a "Wilderness" start). I wouldn't mind putting it back if playtesting proves it's too difficult now, but it's just something I noticed specific to playing the "Compromized Shelter" scenario. I'd like players to have to either run for the door or fight their way out and earn the point the scenario gives them in most circumstances.

Reply:
The window exists on the other shelters though and shouldn't just be removed here. A good alternative might be to make this window (and maybe others randomly) barricaded instead.

Reply:
Yeah, I agree. That's a much more natural solution.

Reply:
I added the window back, and defined a terrain array to account for randomized boarded windows.